admin/css/debug-toolbar/toolbar.scss

```diff
@@ -326,6 +326,23 @@
 
 			&:first-child {
 				border-left: 0;
+				max-width: none;
```

Comment:
little request. Since we're moving to spaces over tabs, can you convert this to spaces also? You can use a separate commit in case others won't like it. 😅